numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -30,6 +33,7 @@ def _get_libdevice_path_decision():
         ("Conda environment (NVIDIA package)", get_nvidia_libdevice_ctk()),
         ("CUDA_HOME", get_cuda_home("nvvm", "libdevice")),
         ("System", get_system_ctk("nvvm", "libdevice")),
+        ("NVIDIA NVCC Wheel", get_libdevice_wheel()),
```

Comment:
Shouldn't we have wheels ahead of system packages? Like with the Debian package, if the wheel comes after the system package, you can never get the toolkit you want if it's different from the system one by installing a wheel.

Reply:
Really good point, will update this.

Reply:
I think this should be ahead of `CUDA_HOME`, too, at the same level as a conda package.

Reply:
@bdice @gmarkall should the search priority be global across all the libraries? If so, https://github.com/NVIDIA/numba-cuda/pull/155/commits/4f686e941e33a2c371dd4a7310cb75a7feaff966 might be a good way forward.

Reply:
Yes it should and we should probably align the search priority with the work @rwgk is doing in cuda-python to build a universal lib finder / path handler.

Reply:
That's the NVIDIA/cuda-python#451 epic.

FYI I took the code from this PR a few weeks ago (NVIDIA/cuda-python#447). Random status update: I added some code on top, that passes all testing for the libraries with cuda-bindings (nvJitLink, nvrtc, nvvm). I'm in the middle of locally testing loading "all" other Nvidia libraries.